doc/nrf/app_dev/device_guides/nrf70/nrf7002eb2_dev_guide.rst

```diff
@@ -106,7 +98,7 @@ Alternatively, add the shield in the project's :file:`CMakeLists.txt` file by us
 
    set(SHIELD nrf7002eb2)
 
-To build for the nRF7002-EB II with nRF54H20 DK, use the ``nrf54h20dk/nrf54h20/cpuapp`` board target with the CMake ``SHIELD`` variable set to ``nrf7002eb2``.
+To build for the nRF7002-EB II with nRF54LM20 DK, use the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target with the CMake ``SHIELD`` variable set to ``nrf7002eb2``.
```

Comment:
I am not sure I like this. We do not build for the shield - we build for the DK, and integrate the shield. I can suggest something below.